libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts

```diff
@@ -39,12 +43,10 @@ import { AboutUsHistoryCarouselComponent } from './components/about-us-history-c
     PrevDirective,
     AboutUsCultureComponent,
     AboutUsPageComponent,
+    AboutUsWeCareComponent,
     AboutUsHistoryCarouselComponent,
     ElewaAboutUsLocationSectionComponent
   ],
-  exports: [
-    TeamMembersCarouselComponent,
-    ElewaAboutUsLocationSectionComponent
-  ],
+  exports: [TeamMembersCarouselComponent, ElewaAboutUsLocationSectionComponent, AboutUsWeCareComponent, ElewaVerticalIconAndTextComponent],
```

Comment:
No need to export the component you created and the verticalIconandtext component does not belong to this module so it should not be exported.